README.md

```diff
@@ -21,12 +21,50 @@ $ cd leaderboard
 
 ```bash
 $ npm install
-$ npm run build
+$ npm start
 ```
 ### Open page in browser
 ```bash
 $ open dist/index.html
 ```
+## How to use API
+
+### Make a new game 
+
+```bash
+  fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games', {
+      method: 'POST',
+      headers: {
+        Accept: 'application/json',
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify({ name: "my new game" }),
+    });
+  };
+```
+### Get data 
+
+```bash
+  fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/idOfgame/scores/')
+      .then((response) => response.json())
+      .then((data) => {
+          console.log(data)
+      });
+
+```
+### Make a new score 
+
+```bash
+ fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/idOfGame/scores/', {
+      method: 'POST',
+      headers: {
+        Accept: 'application/json',
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify({ user: "name Of user", score: "score of user" }),
+    });
+
+```
```

Comment:
- [ ] [OPTIONAL] You can omit this part, remember that the README file is for everyone, this means developers and non-developers, the last Code Reviewer was asking for special instructions about the end-points or if the API needed some kind of key. But he didn't explain himself very well, so don't worry.

Reply:
I understand, thank you very much. 😃